src/test/ui/c-variadic/variadic-ffi-1.stderr

```diff
@@ -28,17 +28,17 @@ error[E0308]: mismatched types
 LL |         let x: unsafe extern "C" fn(f: isize, x: u8) = foo;
    |                                                        ^^^ expected non-variadic fn, found variadic function
    |
-   = note: expected type `unsafe extern "C" fn(isize, u8)`
-              found type `unsafe extern "C" fn(isize, u8, ...) {foo}`
+   = note: expected fn pointer `unsafe extern "C" fn(isize, u8)`
+                 found fn item `unsafe extern "C" fn(isize, u8, ...) {foo}`
```

Comment:
This output is slightly worrying to me. It might actually confuse people more than it clarifies (unless we decide to add another note explaining that fn pointers and fn items can be interchangeable as long as the share the same signature).